package.json

```diff
@@ -18,10 +18,11 @@
     "eslint": "eslint \"**/*.{js,ts}\" --ext .js,.ts --fix",
     "tsc": "tsc",
     "lint": "run-s prettier:* eslint tsc",
-    "test:ci": "JEST_JUNIT_OUTPUT_DIR='./coverage' jest --ci --runInBand --reporters=default --reporters=jest-junit",
+    "test:ci": "JEST_JUNIT_OUTPUT_DIR=\"./coverage\" jest --ci --runInBand --reporters=default --reporters=jest-junit",
     "test:coverage": "cat ./coverage/lcov.info | coveralls",
     "test:watch": "jest --watchAll",
     "test": "jest",
+    "watch": "rollup -c --watch",
```

Comment:
This works, but right now it's a bit slow. That's mostly because of adding a bunch of on-demand Babel polyfills for browser support (currently supporting https://browserl.ist/?q=%3E+0.5%25%2C+last+2+versions%2C+Firefox+ESR%2C+not+dead), and also less so because of minification. I decided for now having dev as close to prod as possible is nice -- but we could always skip those transforms in dev if we prefer.